apps/verification-portal/src/routes/assets/[assetAddress]/+page.server.ts

```diff
@@ -59,6 +59,13 @@ export async function load(event) {
       );
       deepData = entityResponse.data;
       deepData.link = verifiedData;
+
+      const { data: stewardResponse } = await getNewsBySteward(
+        verifiedData.steward_id,
```

Comment:
@smakhtin yep it's hard to test if only the test steward has data, hence why I shared the previous comit iteration.
An option would be to set feature flag workaround (that works on server) to hard cord the test steward id here.

Reply:
I can link the steward to it, no prob. Will do it on Monday.